# This file created by conda-build 25.4.2
# meta.yaml template originally from:
# /home/conda/recipe_root, last modified Sun May 11 07:06:46 2025
# ------------------------------------------------

package:
  name: exceptiongroup
  version: 1.3.0
source:
  sha256: b241f5885f560bc56a59ee63ca4c6a8bfa46ae4ad651af316d4e81817bb9fd88
  url: https://pypi.org/packages/source/e/exceptiongroup/exceptiongroup-1.3.0.tar.gz
build:
  noarch: python
  number: '0'
  script: /home/conda/feedstock_root/build_artifacts/exceptiongroup_1746947292760/_h_env_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placeh/bin/python
    -m pip install . -vv
  string: pyhd8ed1ab_0
requirements:
  host:
    - _libgcc_mutex 0.1 conda_forge
    - _openmp_mutex 4.5 2_gnu
    - bzip2 1.0.8 h4bc722e_7
    - ca-certificates 2025.4.26 hbd8a1cb_0
    - flit-core 3.12.0 pyhd8ed1ab_0
    - flit-scm 1.7.0 pyhd8ed1ab_1
    - importlib-metadata 8.6.1 pyha770c72_0
    - ld_impl_linux-64 2.43 h712a8e2_4
    - libexpat 2.7.0 h5888daf_0
    - libffi 3.4.6 h2dba641_1
    - libgcc 15.1.0 h767d61c_2
    - libgcc-ng 15.1.0 h69a702a_2
    - libgomp 15.1.0 h767d61c_2
    - liblzma 5.8.1 hb9d3cd8_1
    - libnsl 2.0.1 hd590300_0
    - libsqlite 3.49.2 hee588c1_0
    - libuuid 2.38.1 h0b41bf4_0
    - libxcrypt 4.4.36 hd590300_1
    - libzlib 1.3.1 hb9d3cd8_2
    - ncurses 6.5 h2d0b736_3
    - openssl 3.5.0 h7b32b05_1
    - packaging 25.0 pyh29332c3_1
    - pip 25.1.1 pyh8b19718_0
    - python 3.9.22 h85ef794_1_cpython
    - readline 8.2 h8c095d6_2
    - setuptools 80.1.0 pyhff2d567_0
    - setuptools-scm 8.3.1 pyhd8ed1ab_0
    - setuptools_scm 8.3.1 hd8ed1ab_0
    - tk 8.6.13 noxft_h4845f30_101
    - tomli 2.2.1 pyhd8ed1ab_1
    - typing-extensions 4.13.2 h0e9735f_0
    - typing_extensions 4.13.2 pyh29332c3_0
    - tzdata 2025b h78e105d_0
    - wheel 0.45.1 pyhd8ed1ab_1
    - zipp 3.21.0 pyhd8ed1ab_1
  run:
    - python >=3.9
    - typing_extensions >=4.6.0
test:
  commands:
    - pip check
  imports:
    - exceptiongroup
  requires:
    - pip
    - python 3.9.*
about:
  dev_url: https://github.com/agronholm/exceptiongroup
  doc_url: https://docs.python.org/3/library/exceptions.html
  home: https://pypi.org/project/exceptiongroup/
  license: MIT and PSF-2.0
  license_file: LICENSE
  summary: Backport of PEP 654 (exception groups)
extra:
  copy_test_source_files: true
  final: true
  flow_run_id: azure_20250511.2.1
  recipe-maintainers:
    - xylar
  remote_url: https://github.com/conda-forge/exceptiongroup-feedstock
  sha: d0a0401338aa6f2085ee2d7fda0091b61937e24c
